zcncore/transaction_mobile.go

```diff
@@ -1105,9 +1104,14 @@ func (t *Transaction) ZCNSCAddAuthorizer(ip AddAuthorizerPayload) (err error) {
 	return
 }
 
+// ConvertToTokenValue converts ZCN tokens to value
+func ConvertToTokenValue(token float64) uint64 {
```

Comment:
sshould be ConvertTokenToSAS, 1 ZCN token = 1e10 SAS.

Reply:
fixed